client/app/modules/test-session-logs/test-session-logs.controller.js

```diff
@@ -39,6 +39,7 @@ const testSessionLogsController = function testsSessionsController(
     return vm;
 
     function init() {
+        [vm.testSession.platformIcon, vm.testSession.platformVersion, vm.testSession.osName] = testsSessionsService.refactorPlatformData(vm.testSession);
```

Comment:
We don't need to convert to lower case `osName`, so remove it from this refactoring action

Reply:
fixed